### DISCUSSION TITLE
Proposal: Add ability to convert a solution to a state

### DISCUSSION THREAD

**Discussion by ryanjoneil** [domain-independent-dp/didp-rs#27] [Ideas]
_I had an email thread with @Kurorororo about this some months back, but then I let the the thread slip._

Nearly every time I use DIDPpy, I end up writing code to convert the final solution into a state. This is what it looks like.

```python
solution = solver.search()

state = model.target_state
for t in solution.transitions:
    state = t.apply(state, model)
```

I think it would be convenient if there were a method that did this for me. Perhaps something like:

```python
solution = solver.search()
state = solution.state(model)
```

**Comment by Kurorororo** [comment]
Hi Ryan,

Thank you for the suggestion! This sounds like a good idea. I would like to implement this feature in the next release.

**Comment by anubhav-cs** [comment]
Created a corresponding issue #28 based on the discussion.
